### ISSUE TITLE
Use of uninitialized variables?

### ISSUE THREAD

**Issue by eyalroz** [NVIDIA/multi-gpu-programming-models#12]
Hello Jiri,

I'm building your examples (though not with your makefiles) , and am getting compilation warnings from GCC 10 about how `calculate_norm` is passed without initialization to `launch_jacobi_kernel` in `mpi_overlap/jacobi.cpp`:
```
bool calculate_norm;
// ... snip ... 
while (l2_norm > tol && iter < iter_max) {
    // ... snip ... 
    if (use_hp_streams) {
        launch_jacobi_kernel(a_new, a, l2_norm_d, (iy_start + 1), (iy_end - 1), nx, 
                             calculate_norm, compute_stream);
    }   
    // ... snip ... 
    calculate_norm = (iter % nccheck) == 0 || (!csv && (iter % 100) == 0); 
    // ... snip ... 
}   
```
something similar is happening in `single_threaded_copy/jacobi.cu`

**Comment by jirikraus** [comment]
Thanks for the report. Indeed `calculate_norm` is uninitialized in the first iteration. Will look into a fix.

**Comment by eyalroz** [comment]
@jirikraus : That might be enough, but I haven't verified it.

(BTW - If you're interested in why I have a copy of your programs in that repository, or in what's it about in general, you're welcome to email me and ask...)

**Comment by jirikraus** [comment]
Fixed with 3a7090ac0499a544ab157a6319af1c56fbba16d5. Closing this.
